src/components/Explore/TokenTable/TokenRow.tsx

```diff
@@ -146,17 +146,17 @@ const ListNumberCell = styled(Cell)`
     display: none;
   }
 `
-const HeaderLabelCell = styled(Cell)<{ sortable: boolean }>`
+const DataCell = styled(Cell)<{ sortable: boolean }>`
   justify-content: flex-end;
   min-width: 80px;
-  padding-right: 8px;
+  user-select: ${({ sortable }) => (sortable ? 'none' : 'unset')};
```

Comment:
this needs to have padding-right, the design isn't centered -- it's right-aligned with padding-right of 8px